### ISSUE TITLE
Commit some nice clients.yaml script

### ISSUE THREAD

**Issue by jhutar** [redhat-performance/satellite-performance#30]
Make sure we do not have any old versions somewhere in the git.

**Comment by jhutar** [comment]
Err, sorry for not making this clear at the first try. Currently we are using this:

```
#!/bin/bash

size=10
tags="untagged,REG,KAT,DOWNGRADE,REM"
resting=0
project='20170228'
mkdir -p $project

function doit() {
  local server=$1
  local server_short=$( echo $server | cut -d '.' -f 1 )
  local target=$2
  local out="$project/output-$server_short.log"
  cat <<EOF | ssh -i conf/id_rsa_perf -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no root@$server &> $out
    #VG=\$( vgs --separator ':' --noheadings | cut -d ':' -f 1 | sed 's/^  //')
    sed -i 's|^.*host_key_checking.*$|host_key_checking = False|' /etc/ansible/ansible.cfg
    for i in \$( seq 1 \$( expr \$( wc -l /root/container-ips.shuffled | cut -d ' ' -f 1 ) / $size ) ); do
      #pool_data=\$( lvdisplay /dev/\$VG/docker-pool | grep 'Allocated pool data' | sed 's/^.*\s\([0-9]\+\)\.[0-9]\+%$/\1/' )
      #pool_meta=\$( lvdisplay /dev/\$VG/docker-pool | grep 'Allocated metadata' | sed 's/^.*\s\([0-9]\+\)\.[0-9]\+%$/\1/' )
      #if [ \$( expr \$pool_data + \$pool_meta ) -gt 95 ]; then
      #  echo "Not enough pool space"
      #  vgdisplay /dev/\$VG
      #  lvdisplay /dev/\$VG/docker-pool
      #  break
      #fi
      log=out-$project-\$( date +%s ).log
      echo "===== START: batch \$i; size $size [\$( date )] =====" | tee -a \$log
      head -n \$( expr \$i \* $size ) /root/container-ips.shuffled | tail -n $size | cut -d ' ' -f 2 > clients.ini
      ansible-playbook --private-key /root/id_rsa_key -f $size -i clients.ini --extra-vars "server=$target" --tags "$tags" clients.yaml 2>&1 | tee -a \$log
      echo "PASS: \$( grep "unreachable=0\s*failed=0" \$log | wc -l ); FAIL: \$( grep "unreachable=.*failed=" \$log | grep -v "unreachable=0\s*failed=0" | wc -l )"
      echo "===== END: batch \$i; size $size [\$( date )] =====" | tee -a \$log
      sleep $resting
    done
EOF
}

doit dockerhost1.example.com capsule1.example.com &
doit dockerhost2.example.com capsule1.example.com &
doit dockerhost3.example.com capsule2.example.com &
...

wait
```

Maybe it could be nicer and more aligned to the overall workflow - ideally it could take data from inventory? Probably it could be ansible playbook? I believe you have enough insight into what we are doing. Just make it nice, usable, consistent, ubercool.

**Comment by jhutar** [comment]
The script should cleanup `/root/.ssh/known_hosts` or `client-scripts.yaml` should configure ansible to ignore know_hosts file (if possible). Just ran into this scenario:

1. have some fun with containers
2. ssh to them manually to workaround or check something, ssh saves client's fingerprint into known_hosts
3. tierdown, tierup => containers recreated, same IPs, different sshd keys
4. running clients.yaml fails, because container fingerprint in known_hosts is now invalid

**Comment by h4xr** [comment]
known_hosts issue fixed in #37

**Comment by jhutar** [comment]
IMO this is done now.
